### PR TITLE
added latest Hbbtagger

### PR DESCRIPTION
**Pull request by cgalloni**
Changed all configs to use lates HbbTag: ak8 jet collection needs to be updated with the flag  config["UpdateJetCollection"] , inserted in all the configuration files.

Ak8 CSV and Hbbtag are updated, see attached pdf: Red old distributions and Black new ones.
[Ak8csv_Rold_Bnew.pdf](https://github.com/UZHCMS/EXOVVNtuplizerRunII/files/455199/Ak8csv_Rold_Bnew.pdf)
[Hbbtag_Rold_Bnew.pdf](https://github.com/UZHCMS/EXOVVNtuplizerRunII/files/455200/Hbbtag_Rold_Bnew.pdf)

### CODE REVIEW

**Comment by ahinzmann** [comment]
reviewing this one

File: Ntuplizer/config_MC.py [hunk 1]
```diff
@@ -288,7 +288,12 @@
   process.genJetsAK8.jetCorrFactorsSource = cms.VInputTag( cms.InputTag("") )
   process.selectedGenJetsAK8 = selectedPatJetsAK8.clone( src = 'genJetsAK8', cut = cms.string('pt > 20') )
 
-################# Prepare recluster jets with b-tagging ######################
+################# Prepare recluster or update jet collection jets with b-tagging ######################
+
+
+from PhysicsTools.PatAlgos.tools.jetTools import updateJetCollection
```
Comment (ahinzmann) [inline Ntuplizer/config_MC.py]: shouldn't this line also appear in the other 3 config files?

**Comment by cgalloni** [comment]
I’m missing miniAOD_v1 config but the others are ok I think. I realized that it didn't take the config in the python dir . I'll update the PR  in 5 min.

**Comment by ahinzmann** [comment]
Can you check if the global tags used for data and MC we are using include the new Hbb training and switch to the latest. I check https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideFrontierConditions but it doesn't tell. does the b-taggin TWikis say?

**Comment by zucchett** [comment]
I don't think that the double b-tagger weights are in the GT - we needed to update the RecoBTag package manually.

**Comment by ahinzmann** [comment]
ok!

**Comment by cgalloni** [comment]
The b-tagging twiki doesn't say anything about the GT.

**Comment by clelange** [comment]
The files are in cms-data - you can update CMSSW to 8_0_19 then you get it for free, otherwise you need to add the file by hand.

**Pull request merged**